dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -1,126 +1,113 @@
 import com.dotmarketing.util.Config;
-import com.dotmarketing.util.ImportExportUtil;
+import com.dotmarketing.util.ImportStarterUtil;
 import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.MaintenanceUtil;
 import com.dotmarketing.util.UtilMethods;
-import com.liferay.portal.model.User;
 import com.liferay.util.FileUtil;
-import com.rainerhahnekamp.sneakythrow.Sneaky;
 import io.vavr.control.Try;
 
 public class DotCMSInitDb {
 
 	@CloseDBIfOpened
 	private static boolean isConfigured () {
 
-		DotConnect db = new DotConnect();
-		db.setSQL("select count(*) as test from inode");
+		return new DotConnect()
+                .setSQL("select count(*) as test from inode")
+                .getInt("test")>0;
 
-		int test = db.getInt("test");
-		return (test > 0);
 	}
 
-	@CloseDBIfOpened
-	public static void InitializeDb() throws DotDataException {
-
-		final boolean configured = isConfigured();
-
-		if (!configured) {
-			if(Config.getBooleanProperty("STARTERSITE_BUILD", true)){
-				Logger.info(DotCMSInitDb.class,"There are no inodes - initializing db with starter site");
-				try {
-					PrintWriter pw = new PrintWriter(new StringWriter());
-					loadStarterSite(pw);
-					Logger.info(DotCMSInitDb.class, pw.toString());
-				} catch (IOException e) {
-					Logger.error(DotCMSInitDb.class, "Unable to load starter site", e);
-				}
-			}else{
-				Logger.info(DotCMSInitDb.class,"There are no inodes - initializing db for first time use");
-				buildDefaultData();
-			}
-		}else {
-			Logger.info(DotCMSInitDb.class,"inodes exist, skipping initialization of db");
-		}
-	}
+    @CloseDBIfOpened
+    public static void InitializeDb() {
 
-	private static void buildDefaultData() throws DotDataException {
-		try {
-			HostAPI hostAPI = APILocator.getHostAPI();
-			LocalTransaction.wrap(() -> PublicCompanyFactory.createDefaultCompany());
-			// Ensures that default groups are set up
-	//		GroupFactory.createDefaultGroups();
-	//		try {
-	//			LayoutFactory.createDefaultLayouts();
-	//		} catch (Exception e) {
-	//			throw new DotDataException(e.getMessage(), e);
-	//		}
-			// Creating the default host
-			User systemUser = APILocator.getUserAPI().getSystemUser();
-			List<Host> hosts = hostAPI.findAll(systemUser, false);
-			if (hosts.size() == 0) {
-				Logger.debug(DotCMSInitDb.class, "Creating Default Host");
-				hostAPI.findDefaultHost(systemUser, false);
-			}
-	        //Create Default Language
-	        APILocator.getLanguageAPI().createDefaultLanguage();
-			// Creating the default content structures if it not exists.
-			StructureFactory.createDefaultStructure();
-		} catch (DotSecurityException e) {
-			Logger.fatal(DotCMSInitDb.class, "Unable to initialize default data", e);
-			throw new DotRuntimeException(e.getMessage(), e);
-		} catch (Exception e) {
-			Logger.fatal(DotCMSInitDb.class, "Unable to initialize default data", e);
-			new DotDataException(e.getMessage(), e);
-		}
-	}
+        if (!isConfigured()) {
```

Comment:
why was the `buildDefaultData`  removed? what would be the behavior if the `STARTERSITE_BUILD` is null?